source/common/common/stack_array.h

```diff
@@ -19,7 +19,7 @@ namespace Envoy {
 // the array won't be deallocated until the function containing the macro returns.
 // We can't call alloca in the StackArray constructor since the memory would
 // be freed when the constructor returns.
-#define STACK_ARRAY(var, type, num) StackArray<type> var(::alloca(sizeof(type) * num), num)
+#define STACK_ARRAY(var, type, num) Envoy::StackArray<type> var(::alloca(sizeof(type) * num), num)
```

Comment:
Hmm...it looks weird that this is in namespace Envoy, but still qualified. Most users of this will be in the same namespace, but I guess that Quic isn't.

I'm OK with this but WDYT of leaving this file alone and using Envoy::STACK_ARRAY(...) at call-sites in quick, so it's clear in the Quic code you are referencing something that's really part of Envoy.